packages/lib-classifier/src/plugins/drawingTools/models/marks/Mark/Mark.spec.js

```diff
@@ -192,5 +198,21 @@ describe('Models > Drawing Task > Mark', function () {
       })
     })
   })
+
+  describe('snapshots', function () {
+    let snapshot
+
+    before(function () {
+      snapshot = getSnapshot(mark)
+    })
+
+    it('should not have an ID', function () {
+      expect(snapshot.id).to.be.undefined()
+    })
+
+    it('should have an annotations array', function () {
+      expect(snapshot.annotations).to.be.a('array')
+    })
+  })
```

Comment:
There should be tests for `applySnapshot` as well, that an id is generated and the annotations array is converted to a map?

Reply:
The preprocessor should already be tested with `Mark.create(snapshot)` but I'll double check.

Reply:
`mark.id` already has a test but `mark.annotations` was only tested for existence. I've updated that to test for type.